WalletWasabi/JsonConverters/Timing/TimeSpanJsonConverterNg.cs

```diff
@@ -0,0 +1,44 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace WalletWasabi.JsonConverters.Timing;
+
+public class TimeSpanJsonConverterNg : JsonConverter<TimeSpan>
+{
+	/// <inheritdoc />
+	public override TimeSpan Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+	{
+		if (reader.TokenType != JsonTokenType.String)
+		{
+			return default;
+		}
+
+		string? stringValue = reader.GetString();
+		return Parse(stringValue);
```

Comment:
What's wrong with `TimeSpan.Parse(stringValue)`?
Does that behave so differently that we need our own parser function?

Reply:
we should look into this in another PR, it's weird